### PR TITLE
[IOTDB-7] Fix OpenFileNumUtilTest

### PR DESCRIPTION
**Pull request by liuruiyiyang**
1.Remove usage of `com.sun.management.UnixOperatingSystemMXBean` and use lsof command for all open file number statistics. 

2.Check the output of lsof command to exam whether the lsof command is valid.

### CODE REVIEW

File: iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java [hunk 1]
```diff
@@ -98,20 +93,21 @@ private static boolean isNumeric(String str) {
...
     Process pro1;
     Runtime r = Runtime.getRuntime();
     String osName = System.getProperty("os.name").toLowerCase();
     try {
       String command;
+      String processName = IOTDB_PROCESS_KEY_WORD;
       if (osName.startsWith(LINUX_OS_NAME)) {
         command = String.format(SEARCH_PID_LINUX, processName);
       } else {
         command = String.format(SEARCH_PID_MAC, processName);
```
[the remark this answers is not in the document]
Comment (liuruiyiyang) [reply iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java]: Windows is not supported.

[the remark this answers is not in the document]
Comment (jixuan1989) [reply iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java]: So why not handle the case....

[the remark this answers is not in the document]
Comment (liuruiyiyang) [reply iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java]: fixed

**Comment by MyXOF** [comment]
Please check travis results frequently and ensure that failure of tests is not caused by OpenFileNumTest. Try to retrigger those failed tests as soon as possible.

**Pull request merged**